engine/apps/api/serializers/user.py

```diff
@@ -140,6 +154,13 @@ def get_cloud_connection_status(self, obj):
             return status
         return None
 
+    def get_is_currently_oncall(self, obj: User) -> bool:
```

Comment:
Wondering if this could affect performance, not sure how often the UI gets the user information.

Reply:
Fair point 👍  It looks like the UI makes a call to `GET /users` on page load on quite a few pages:
- users
- escalation chains
- schedules
- schedule detail

Alternatively, I could add support for a new query param, `verbose`, which conditionally changes the serializer used for `GET /users` + make a new serializer which subclasses `UserSerializer` and adds `is_currently_oncall`.

(without any meaningful amount of data locally, it's hard to say what the performance impact would be, I'd probably have a better idea once this is rolled out to staging).

Reply:
addressed by adding support for a `short` query param (which defaults to `true`). When `false` will use a different response serializer which contains this field.